test/unit/gateways/nuvei_test.rb

```diff
@@ -149,6 +151,19 @@ def test_successful_stored_credentials_merchant_recurring
     end.respond_with(successful_purchase_response)
   end
 
+  def test_successful_authorize_bank_account
```

Comment:
💬 Comment:

It's not a big issue, bu what's the purpose of setting and instance variable @bank_account in the before block if it will just be used once?

Reply:
Yes, it's because the bank account can be used with **payouts**, so we will eventually add more tests for the bank account